project.html

```diff
@@ -0,0 +1,277 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Portfolio: Ademola Adebayo</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="project.css">
+</head>
+
+<body>
+  <header class="main">
+    <nav class="navbar">
+      <div class="nav-left">
+        <div>
+          <h2>Smart City</h2>
+        </div>
+      </div>
+      <div class="nav-right">
+        <ul class="nav-links">
+          <li><a class="links" href="#portfolio">Portfolio</a></li>
+          <li><a class="links" href="#about">About</a></li>
+          <li><a class="links" href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] Nice work Team, But currently your mobile menu does not look exactly like the design in figma, kindly fix this
    your design 👇 
     
![image](https://user-images.githubusercontent.com/66097371/163026654-a0038fd1-bb50-4183-80f3-d1d1cf7ee185.png)
figma design 👇 
  
![image](https://user-images.githubusercontent.com/66097371/163027099-ba93f5cc-e52e-4a75-b614-aecb6f2d2f85.png)